docs/docsite/rst/porting_guides/porting_guide_2.9.rst

```diff
@@ -583,6 +583,11 @@ Removed Lookup Plugins
 
 * ``redis_kv`` use :ref:`redis <redis_lookup>` instead.
 
+Noteworthy plugin changes
```

Comment:
Did this actually happen in 2.9 (since it's already branched) or does this change need to go into `porting_guide_2.10.rst`?